search/src/main/scala/weco/api/search/services/ImagesService.scala

```diff
@@ -53,7 +54,16 @@ class ImagesService(
         requestBuilder.requestWithSimilarColors
     }
 
-    val searchRequest = builder(index, imageId, nVisuallySimilarImages)
+    val defaultMinScore: Double = similarityMetric match {
+      case SimilarityMetric.Blended  => 300
+      case SimilarityMetric.Features => 300
+      case SimilarityMetric.Colors   => 20
+    }
```

Comment:
Can you explain how you came up with these default scores? Why is `Colors` so much lower?

Reply:
Sure. There's [a notebook in the data science repo](https://github.com/wellcomecollection/data-science/blob/main/notebooks/similarity_problems/notebooks/01-similarity-scores.ipynb) which produced the results in https://github.com/wellcomecollection/platform/issues/5581. Based on that analysis, we decided that ~300 seemed like an appropriate threshold for the blended similarity metric.

I re-ran that analysis with the `state.inferredData.lshEncodedFeatures` and `state.inferredData.palette` fields individually, and produced these corresponding graphs:

**lshEncodedFeatures:**
![2c12fc1b-c57d-490f-83fd-7590149831de](https://user-images.githubusercontent.com/11006680/184423014-d9a17c96-ed74-4565-8680-0f7e61ef1300.png)

**palette:**
![9c4a9ae3-49d1-4834-a769-f56f32e1bc1c](https://user-images.githubusercontent.com/11006680/184423004-ded05145-5f19-4bb8-8701-1b50d2df985a.png)

If my mental maths is right, the scores for colours are generally lower because the `state.inferredData.palette` field contains fewer, more commonly occurring terms.

Reply:
Ah, super. Maybe just include a comment pointing to that ticket, so we can find this again in future?